api/output_policy.go

```diff
@@ -0,0 +1,197 @@
+package api
+
+import (
+	"bytes"
+	"fmt"
+	"regexp"
+	"strings"
+
+	retryablehttp "github.com/hashicorp/go-retryablehttp"
+	"github.com/hashicorp/vault/sdk/helper/jsonutil"
+)
+
+const (
+	ErrOutputPolicyRequest = "output policy request"
+)
+
+var LastOutputPolicyError *OutputPolicyError
```

Comment:
I think I'll be leaving this for a future bigger refactor as well because currently this variable is relied on by both the api and command packages for the way these flags work. 

At least the docstrings for the OutputCurlString field on the Client struct already says `Note: It is not thread-safe to set this and make concurrent requests with the same client.` which at least maybe helps people be aware of this generally.